app/assets/stylesheets/frontend.less

```diff
@@ -16,34 +16,43 @@ body {
 	font-family: sans-serif;
 }
 
-.section_headings {
-    position: relative;
-    z-index: 1;
-    overflow: hidden;
-    text-align: center;
+h1 {
+	/* Page itself has top padding */
+	margin-top: 0
 }
 
-.section_headings:before, .section_headings:after {
-    position: absolute;
-    top: 51%;
-    overflow: hidden;
-    width: 50%;
-    height: 1px;
-    content: '\a0';
-    background-color: black;
+h1, h2 {
+	margin-bottom: 20px;
 }
 
-.section_headings:before {
-    margin-left: -51%;
-    text-align: right;
+h2 {
+	border-top: 1px solid #dddddd;
+	padding-top: 8px;
+	page-break-before: always
 }
 
-.section_headings:after {
-    margin-left: 1%;
+h1 + h2 {
+	page-break-before: avoid
 }
 
-h1, h2 {
-    margin-bottom: 20px;
+h1, h2, h3, h4, h5, h6 {
+	page-break-after: avoid
+}
+
+h3 {
+	border-top: 1px solid #eeeeee;
```

Comment:
@pwithnall , from actually reading your first link:

```
Centered text is best used for headlines and short lines of text. Users can read them with ease because the lines are short, scannable and don’t need repeated eye movements. They can also give your layout a balanced look with its symmetrical format.
```